experiments/functional/snapshot_deletion/jiva/test_replica_restart.yml

```diff
@@ -0,0 +1,23 @@
+- name: Get the name of replica pods
+  shell: >
+    kubectl get pods -l openebs.io/replica=jiva-replica,openebs.io/persistent-volume={{ pv_name }} -n {{ app_ns }} --sort-by=.metadata.creationTimestamp -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}}'
+  args:
+    executable: /bin/bash
+  register: replica_pod
+
+- name: Set the replica pod names to variables
+  set_fact:
+    replica_name_three: "{{ replica_pod.stdout_lines[2].split()[0] }}"
+
+- name: Delete replica with latest creation timestamp
+  shell: >
+    kubectl delete pod {{ replica_name_three }} -n {{ app_ns }}
+  args:
+    executable: /bin/bash
+
+- name: Pause for some time for a new replica to come up
```

Comment:
There should be comment explaining the reason for this pause.

Reply:
sounds good